src/smolvm/vm.py

```
@@ -2123,6 +2184,68 @@ async def _async_wait_for_process(self, pid: int, timeout: float) -> None:
+        logger.debug("Async started libkrun: PID=%d, vm_id=%s", process.pid, vm_info.vm_id)
+
+        async def _close_log_when_done() -> None:
+            try:
+                await process.wait()
+            finally:
+                try:
+                    log_file.close()
+                finally:
+                    self._log_files.pop(key, None)
+
+        asyncio.create_task(_close_log_when_done())
```

Ruff (0.15.9)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)